adaptive_scheduler/kernel/fullscheduler_ortoolkit.py

```diff
@@ -201,8 +201,8 @@ def schedule_all(self, timelimit=0):
 
         # Objective: Maximize the merit functions of all scheduled requests (eq 1);
         objective = solver.Maximize(solver.Sum(
-            [isScheduled * (priority + (0.1 / (winidx + 1.0))) for req, winidx, priority, resource, isScheduled in
-             requestLocations]))
+            [isScheduled * priority for _, _, priority, _, isScheduled in requestLocations])
```

Comment:
We used to add the weighting toward earlier windows in the solver maximization here, but I've moved it to incorporate it into the priority when building the kernel input, since airmass will do something different.